components/disk/component.go

```diff
@@ -87,6 +92,23 @@ func New(gpudInstance *components.GPUdInstance) (components.Component, error) {
 	}
 	c.mountPointsToTrackUsage = muntPointsToTrackUsage
 
+	if gpudInstance.EventStore != nil && runtime.GOOS == "linux" {
+		var err error
+		c.eventBucket, err = gpudInstance.EventStore.Bucket(Name)
+		if err != nil {
+			ccancel()
+			return nil, err
+		}
+
+		if os.Geteuid() == 0 {
```

Comment:
We have required running gpud with root permission, so shall we remove the condition?

Reply:
Oh let us keep it for now, to make ci pass